src/wild_pokemon_area.c

```diff
@@ -265,7 +265,7 @@ static bool32 PokemonInAnyEncounterTableInMap(const struct WildPokemonHeader * d
         return TRUE;
     if (PokemonInEncounterTable(data->waterMonsInfo, species, 5))
         return TRUE;
-    if (PokemonInEncounterTable(data->fishingMonsInfo, species, 12))
+    if (PokemonInEncounterTable(data->fishingMonsInfo, species, 12)) // 10
```

Comment:
What does this comment mean?

Reply:
It's supposed to be 10 but the developer mistakenly put 12 here. This causes the game to think there's two entries more than there actually are when looking up a Pokemon's nest. By dumb luck, these two entries are 0 padding and the high short of the pointer to the fishing encounters array, neither of which is a valid species identifier, so the player sees no ill effect.